wintun-testing/cziti/sdk.go

```diff
@@ -131,12 +132,12 @@ func zitiError(code C.int) error {
 	return errors.New(C.GoString(C.ziti_errorstr(code)))
 }
 
-func LoadZiti(cfg string) (*CZitiCtx, error)  {
+func LoadZiti(cfg string) (*CZitiCtx, error) {
 	ctx := CZitiCtx{}
 	ctx.options.config = C.CString(cfg)
 	ctx.options.init_cb = C.nf_init_cb(C.initCB)
 	ctx.options.service_cb = C.nf_service_cb(C.serviceCB)
-	ctx.options.refresh_interval = C.int(10)
+	ctx.options.refresh_interval = C.int(600)
```

Comment:
this needs a 'todo' or to be configurable right?

Reply:
configurable probably until we get identity configs from controller